hyperspy/io_plugins/msa.py

```diff
@@ -216,34 +212,14 @@ def parse_msa_string(string, filename=None):
                     mapped.set_item(keywords[clean_par]['mapped_to'] +
                                     '_units', units)
 
-    # The data parameter needs some extra care
```

Comment:
I think that most of the removed code was actually necessary. The msa data and time format follows US conventions, and that's why the locale was set temporarily to US (there may be a better way of achieving this though). However, there are a lot of msa files out there with date and time in the wrong format. Therefore, if the parsing is not inside a try, except, the reader may raise an exception for many files that contain valid data but invalid date.